activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/JsonSchemaModelValidator.java

```diff
@@ -66,50 +68,55 @@ public void validate(byte[] bytes,
             throw new SemanticModelValidationException(validationException.getMessage(),
                                                        getValidationErrors(validationException, processExtensionJson));
```

Comment:
Shouldn't this one return the list of errors stead of throwing a SemanticModelValidationException?